README.md

```diff
@@ -92,102 +93,93 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.89 - February 2025 Update
+### 0.90 - March 2025 Update
 
-In this release, we focused on new features, stability, accessibility and automation.
+In this release, we focused on new features, stability, and automation.
 
 **✨Highlights**
 
- - Enhanced Advanced Paste by adding media transcoding support to convert different video and audio file formats! Thanks [@snickler](https://github.com/snickler) for your help!
- - Fixed crashes when loading thumbnails after the .NET 9 update and resolved PowerLauncher.exe blocking other MSI installers from creating shortcuts!
- - Fixed accessibility issues across FancyZones, Image Resizer, and Settings to improve screen reader support and clarity!
- - Enhanced UI automation framework across modules and added new tests to cover manual checks, with more improvements coming!
+ - New module: Command Palette ("CmdPal") -  A fully extensible quick launcher with a richer display and additional capabilities without sacrificing performance, allowing you to start anything with the shortcut **Win+Alt+Space**! Thanks [@zadjii-msft](https://github.com/zadjii-msft), [@niels9001](https://github.com/niels9001), [@joadoumie](https://github.com/joadoumie), [@ethanfangg](https://github.com/ethanfangg) and [@krschau](https://github.com/krschau)!
```

Comment:
oh my god, I only added commit authors to the original PR. Connor absolutely needs to be on this list too. 

```suggestion
 - New module: Command Palette ("CmdPal") -  A fully extensible quick launcher with a richer display and additional capabilities without sacrificing performance, allowing you to start anything with the shortcut **Win+Alt+Space**! Thanks [@zadjii-msft](https://github.com/zadjii-msft), [@niels9001](https://github.com/niels9001), [@joadoumie](https://github.com/joadoumie),  [@plante-msft](https://github.com/plante-msft), [@ethanfangg](https://github.com/ethanfangg) and [@krschau](https://github.com/krschau)!
```